x-pack/plugins/graph/public/apps/listing_route.tsx

```diff
@@ -26,6 +26,7 @@ type GraphUserContent = UserContentCommonSchema;
 const toTableListViewSavedObject = (savedObject: GraphWorkspaceSavedObject): GraphUserContent => {
   return {
     id: savedObject.id!,
+    namespaces: savedObject.namespaces ?? [],
```

Comment:
I think `namespaces` should always be present on the saved object, so we could also add an assertion operator rather than coalescing to an empty array. WDYT?